npm/webpack-dev-server/package.json

```diff
@@ -42,7 +42,8 @@
     "ts-node": "^10.9.2",
     "webpack": "npm:webpack@^5",
     "webpack-4": "npm:webpack@^4",
-    "webpack-dev-server-3": "npm:webpack-dev-server@^3"
+    "webpack-dev-server-3": "npm:webpack-dev-server@^3",
+    "webpack-dev-server-5": "npm:webpack-dev-server@^5"
```

Comment:
Out of curiosity, why isn't `npm:webpack-dev-server@v4` listed?

Reply:
because its installed under `webpack-dev-server` since its the default that ships with `@cypress/webpack-dev-server`. Eventually we want to bump it to 5, but that is a breaking change